src/components/Layout/Header/index.tsx

```diff
@@ -1,16 +1,25 @@
 import { Box, IconButton, Typography } from "@mui/material";
-import { headerStyles } from "./headerStyles";
 import AddIcon from "@mui/icons-material/Add";
+import { Style } from "shared/theme";
 
-export const Header = () => {
-  return (
-    <Box sx={headerStyles.header}>
-      <Typography variant="h3" component="h1" color="white">
-        Product Library
-      </Typography>
-      <IconButton>
-        <AddIcon color="primary" />
-      </IconButton>
-    </Box>
-  );
+const headerStyles: Style = {
```

Comment:
There is no `sx` interface on MUI?

Reply:
there is but its for a single element, i created a new one so i can define all the styles in one object instead of inline

Reply:
Let's talk about it